app/views/homepage/_search_bar.haml

```diff
@@ -1,3 +1,10 @@
+- # report search usage: keyword search results fetched
+- content_for :extra_javascript do
+  :javascript
+    if(document.getElementById('q').value !== "" && document.getElementById('lc') == null) {
+      window.report_analytics_event('user', "main search", "keyword search");
+    }
+
```

Comment:
They are used independently and feature flags doesn't reflect that (:location_search, :new_search & with keyword search it doesn't matter if we use search-api or sphinx). The idea was to just collect info to GA about how much we get search queries in different marketplaces. First to figure out location search usage, but later (when both options are visible) we could maybe use that info to differentiate marketplaces.

We could move this to index.haml to cover both cases, but I felt that it's better to reference input-elements inside the partials where they are defined.